neo4j-examples/paradise-papers-django
# Display info in tables according to data from

API

Closes [Ticket](https://trello.com)






## Testing

Steps to manually verify the change:
1. _Setup_: open `paradise-papers-django/search_frontend/index.html`
2. _Exercise_: click search. 
3. _Verify_: check if information through tabs are dynamically. 
PS: only the information, not the header text 👍 
ex : 
![image](https://user-images.githubusercontent.com/29434543/34907535-8f267d76-f856-11e7-9310-c8377a266be3.png)

![image](https://user-images.githubusercontent.com/29434543/34907536-9800a34a-f856-11e7-8a0c-ae9bb9ea6777.png)

![image](https://user-images.githubusercontent.com/29434543/34907537-a2816c00-f856-11e7-8953-b1ee6f9af8b7.png)
